.travis.yml

```diff
@@ -64,6 +70,7 @@ notifications:
 before_install:
 - pip install codecov
 - pip install numpy==1.6.1
+- pip install pyyaml
```

Comment:
Ah, good catch. I've added this to the installation instructions -- previously, the "yaml" that was being used as a dependency was the one from the GAE third-party libraries, but build.py does not make use of that.